src/pages/ReimbursementAccount/VerifyIdentity/VerifyIdentity.tsx

```diff
@@ -1,51 +1,54 @@
-import PropTypes from 'prop-types';
 import React, {useCallback} from 'react';
 import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
 import {withOnyx} from 'react-native-onyx';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useLocalize from '@hooks/useLocalize';
 import useSubStep from '@hooks/useSubStep';
+import type {SubStepProps} from '@hooks/useSubStep/types';
 import useThemeStyles from '@hooks/useThemeStyles';
-import {reimbursementAccountPropTypes} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
-import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
-import getDefaultValueForReimbursementAccountField from '@pages/ReimbursementAccount/utils/getDefaultValueForReimbursementAccountField';
+import {reimbursementAccountDefaultProps} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
 import * as BankAccounts from '@userActions/BankAccounts';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount} from '@src/types/onyx';
+import type {OnfidoData} from '@src/types/onyx/ReimbursementAccountDraft';
 import OnfidoInitialize from './substeps/OnfidoInitialize';
 
-const propTypes = {
+type VerifyIdentityOnyxProps = {
     /** Reimbursement account from ONYX */
-    reimbursementAccount: reimbursementAccountPropTypes,
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+};
 
+type VerifyIdentityProps = VerifyIdentityOnyxProps & {
     /** Goes to the previous step */
-    onBackButtonPress: PropTypes.func.isRequired,
+    onBackButtonPress: () => void;
 
     /** Exits flow and goes back to the workspace initial page */
-    onCloseButtonPress: PropTypes.func.isRequired,
-};
-
-const defaultProps = {
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
+    onCloseButtonPress: () => void;
 };
 
-const bodyContent = [OnfidoInitialize];
+const bodyContent: Array<React.ComponentType<SubStepProps>> = [OnfidoInitialize];
 
-function VerifyIdentity({reimbursementAccount, onBackButtonPress, onCloseButtonPress}) {
+function VerifyIdentity({reimbursementAccount = reimbursementAccountDefaultProps, onBackButtonPress, onCloseButtonPress}: VerifyIdentityProps) {
```

Comment:
Do we really need a default value for `reimbursementAccount`? In which case will it be applied? I guess `reimbursementAccount` can be or `null` or the actual `reimbursementAccount` value